### PR TITLE
[Fix][TVMScript]TVMScript BinOP printing refactor 

### PR DESCRIPTION
**Pull request by cyx-6**
This PR fixes the output for `T.Div(int, int)`. It will print `T.Div(int, int)`, instead of `int / int`, to avoid the integer division ambiguity in parser.

And this PR refactors the logic of binary operators printing in TVMScript. The updated TVMScript printer will print the binary operator to avoid constant folding when parsing back.

### CODE REVIEW

**Comment by junrushao** [comment]
Any updates?

File: python/tvm/script/parser/tir/operation.py [hunk 1]
```diff
@@ -46,12 +46,12 @@ def r(op: Type, i: int, m: OpMethod):  # pylint: disable=invalid-name
 
     for i in [0, 1]:
         # Case 1. binop
```
Comment (junrushao) [inline python/tvm/script/parser/tir/operation.py:48]: i was thinking if we could remove all the binary operators here given our printer has taken good care of it

Comment (cyx-6) [reply python/tvm/script/parser/tir/operation.py:48]: Sure! All those overloaded op have been removed.

**Review by junrushao — APPROVED** [review]
LGTM!

**Comment by junrushao** [comment]
Feel free to merge it yourself as soon as it turns green

**Pull request merged**